go.mod

```diff
@@ -172,7 +172,7 @@ require (
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c // indirect
 	github.com/prometheus/client_model v0.6.1 // indirect
-	github.com/prometheus/common v0.54.0 // indirect
+	github.com/prometheus/common v0.55.0 // indirect
```

Comment:
I suspect this doesn't affect our ability to drop the `replace` for this module?

Reply:
We could drop the `replace`, but as soon as it moves past v0.47.0 we run into 

```
github.com/prometheus/client_golang/prometheus/testutil/promlint
../../go/pkg/mod/github.com/prometheus/client_golang@v1.18.0/prometheus/testutil/promlint/promlint.go:71:38: undefined: expfmt.FmtText
```

and removing that `replace` just kicks the same error over to component-base. I think it's fixed in component-base v0.31.0 but that's jumping ahead of other k8s vendoring. So it's probably safest to leave the `replace`s intact for now.

Reply:
Cool, that's what I was expecting. Thanks for checking!